### DISCUSSION TITLE
Is there some kind of process for PR's?

### DISCUSSION THREAD

**Discussion by EleotleCram** [espressif/arduino-esp32#8189] [General]
Hi guys,

A few days ago I submitted this PR: https://github.com/espressif/arduino-esp32/pull/8166

As it is my first PR, there seems some anti-spam protection that got activated or something. I've waited a few days, but I don't see any activity on it. Is there some process that I'm missing here, to get some attention to it?

I've put effort into this contribution, I'd hate to see this effort go to waste because I'm missing understanding of how the PR process for this project works. Can anybody shed some light on this, please? (That would help a lot.)

Kind regards,

Marcel

**Comment by mrengineer7777** [comment]
Hi Marcel. It takes patience and prayers. New features take 1-6 months to be reviewed and merged. Maybe less for a nice quality example like yours. I took a quick glance and it seems well done.
I know right now the devs are working on massive updates for version 3.0.0.

**Comment by VojtechBartoska** [comment]
Hi, reviewers added. We'll check your PR soon.

**Comment by EleotleCram** [comment]
Hi guys, thanks for the replies.

I can be patient, no rush. I guess it's more the not knowing what will happen next/wondering if it will be picked up in due time and not overlooked, that made ask the question.

Good thing is that this question/answers are now available for future PR contributors.

Kind regards,

Marcel

**Reply by VojtechBartoska** [reply]
Actually it's good feedback for us, we should at least notify you that we are aware of the PR and we'll look on it a bit later. Thanks for your patience and also contribution!
